docs/integrations/azure/locations.md

```diff
@@ -0,0 +1,30 @@
+---
+id: locations
+title: Azure DevOps Locations
+sidebar_label: Locations
+description: Documentation on Azure DevOps location integration
+---
+
+The Azure integration supports loading catalog entities from Azure DevOps.
+Components can be added to
+[static catalog configuration](../../features/software-catalog/configuration.md),
+or registered with the
+[catalog-import](https://github.com/backstage/backstage/tree/master/plugins/catalog-import)
+plugin.
```

Comment:
is it useful to mention HOW the registration happens from an end user perspective (as in, with the Create function in the sidebar ... if the org hasn't chosen to rename it -_-) instead?

Reply:
Mmm, not sure I agree. This is sort of the discussion of documentation for different user groups. It's presently aimed at the person setting up Backstage, who would want to know the name of the plugin to add if it's missing. I think it's okay this way for now.